doc/interop-test-descriptions.md

```diff
@@ -543,21 +552,15 @@ Asserts:
 
 ### unimplemented_method
 
-Status: Not yet implementable
+Status: Ready for implementation. Blocking beta.
 
-This test verifies calling unimplemented RPC method returns unimplemented
-status.
+This test verifies calling unimplemented RPC method returns the UNIMPLEMENTED status code.
 
 Procedure:
-* Client calls UnimplementedCall with:
+* Client calls UnimplementedCall with an empty request (defined as `grpc.testing.Empty`):
```

Comment:
Since this isn't the normal test service, it does seem good to note the service name.